tests/framework/utils.py

```diff
@@ -760,16 +762,38 @@ def start_screen_process(screen_log, session_name, binary_path, binary_params):
         delay=1,
     ).group(1)
 
-    binary_clone_pid = int(
-        open("/proc/{0}/task/{0}/children".format(screen_pid), encoding="utf-8")
-        .read()
-        .strip()
-    )
-
     # Configure screen to flush stdout to file.
     flush_cmd = 'screen -S {session} -X colon "logfile flush 0^M"'
     run_cmd(flush_cmd.format(session=session_name))
```

Comment:
Maybe we could capture the output / error code here, and add it to the raised exceptions, to shed some light on why it fails.

Reply:
I don't see why not. It is currently being deliberately ignored, I'm not 100% sure why, but I will tweak it and see how CI does.